### DISCUSSION TITLE
如何让致谢、摘要等标题居中并和正文一样拥有页眉

### DISCUSSION THREAD

**Discussion by ZeallYang** [TheNetAdmin/zjuthesis#144] [General]
如题，求助如何让致谢、摘要等标题居中，另外希望这些页面和正文一样拥有页眉

**Comment by TheNetAdmin** [comment]
什么年级什么专业的模板

**Reply by ZeallYang** [reply]
博士论文控制学院的模板，旧版本下居中问题是这么修改的，https://github.com/TheNetAdmin/zjuthesis/blob/42fec39181ff128e128dd31bc7ea10f57c77fe8c/config/commands.tex#L97-L115

前几天版本更新之后找不到在哪里修改了。

另外就是致谢、摘要等页面上增加页眉的问题，望指教。

**Reply by TheNetAdmin** [reply]
标题格式可以临时把这里

https://github.com/TheNetAdmin/zjuthesis/blob/8da2f128ec4fb1c6255b406811fd28ed52c2f7cf/config/commands.tex#L115

改成这样

https://github.com/TheNetAdmin/zjuthesis/blob/8da2f128ec4fb1c6255b406811fd28ed52c2f7cf/config/commands.tex#L43-L46

**Reply by TheNetAdmin** [reply]
页眉页脚改这里

https://github.com/TheNetAdmin/zjuthesis/blob/8da2f128ec4fb1c6255b406811fd28ed52c2f7cf/config/format/general/layout.tex#L42-L61

参考其上方的 undergraduate 设定
